Library/Formula/libfuse.rb

```diff
@@ -0,0 +1,25 @@
+class Libfuse < Formula
+  desc "Reference implementation of the Linux FUSE (Filesystem in Userspace) interface."
+  homepage "https://github.com/libfuse/libfuse"
+  url "https://github.com/libfuse/libfuse/archive/fuse_2_9_5.tar.gz"
+  sha256 "ccea9c00f7572385e9064bc55b2bfefd8d34de487ba16d9eb09672202b5440ec"
+  head "https://github.com/libfuse/libfuse.git"
```

Comment:
Please add
```ruby
# tag "linuxbrew"
```
I use this comment for formula that are present in Linuxbrew but not in Homebrew.